composer.json

```diff
@@ -56,8 +56,9 @@
     "oomphinc/composer-installers-extender": "^1.1"
   },
   "require-dev": {
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
     "phpcompatibility/phpcompatibility-wp": "^2.1",
-    "phpunit/phpunit": "<6.0",
+    "phpunit/phpunit": "<8.0",
```

Comment:
Will have to check, but we might need to use the same phpunit version that WP Core uses. This is ok for now.

Reply:
Core is going to use: "phpunit/phpunit": "^5.0 || ^6.0 || ^7.0"